pyramid/config/views.py

```diff
@@ -1545,11 +1588,16 @@ def notfound(request):
         being used, :class:`~pyramid.httpexceptions.HTTPMovedPermanently will
         be used` for the redirect response if a slash-appended route is found.
 
-        .. versionchanged:: 1.6
         .. versionadded:: 1.3
+
+        .. versionchanged:: 1.6
```

Comment:
I pushed changes to master, 1.6-branch and 1.7-branch to fill in this version changed information. It is not yet merged into this branch.